spec/requests/api/v1/users_request_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+
+RSpec.describe "Api::V1::Users", type: :request do
+  include_context "with API request"
+
+  describe "POST /api/v1/users" do
+    let(:do_request) do
+      post api_v1_users_url, params: { user: user_attributes }
+    end
+
+    let(:user_attributes) { attributes_for(:user) }
```

Comment:
Better to write attributes explicitly
```suggestion
    let(:user_attributes) { { full_name: "John Smith", email: "john.smith@example.com" } }
```